src/components/kytos/misc/StatusMenu.vue

```diff
@@ -0,0 +1,764 @@
+                        <input v-model="switchTextFilter[2][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="switchTextFilter[3][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="switchTextFilter[4][1]"></input>
+                      </th>
+                    </tr>
+                  </thead> 
+                  <tbody>
+                    <tr v-for="(item, index) in filtered_switchData" :key="item.dpid">
+                      <td><k-button class="statusTableButton" :icon="isStoredS(item, index)" v-on:click.native="add_remove_Switches(item, index)"></k-button></td>  
+                      <td v-on:click="show_infoPanel(item.dpid)">{{item.name}}</td>
+                      <td :class="statusColor(item)">{{item.status}}</td>
+                      <td>{{item.status_reason | splitStatusReasons}}</td>
+                      <td>{{item.enabled}}</td>
+                      <td>{{item.active}}</td>              
+                    </tr>
+                  </tbody>             
+              </table>
+            </div>
+          </k-accordion-item>
+          <k-accordion-item title="Links">
+            <k-accordion-item title="Filters">
+            <k-button-group>
+              <k-button :class="selectedLinkData('ORIGINAL')" icon="database" title="Show-All" tooltip="Show-All" v-on:click.native="changeLinkData('ORIGINAL')"></k-button>
+              <k-button :class="selectedLinkData('PRE_SELECTED')" icon="clipboard-check" title="Show-Selected" tooltip="Show-Selected" v-on:click.native="changeLinkData('PRE_SELECTED')"></k-button>
+              </k-button-group>
+            </k-accordion-item>
+            <div class="data_status_table">
+              <table>
+                  <thead>
+                    <tr>
+                      <th rowspan="2"></th>
+                      <th>Name</th>
+                      <th>Status</th>
+                      <th>Reason</th>
+                      <th>Enabled</th>
+                      <th>Active</th>
+                    </tr>
+                    <tr>
+                      <th>
+                        <input v-model="linkTextFilter[0][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="linkTextFilter[1][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="linkTextFilter[2][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="linkTextFilter[3][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="linkTextFilter[4][1]"></input>
+                      </th>
+                    </tr>
+                  </thead> 
+                  <tbody>
+                    <tr v-for="(item, index) in filtered_linkData" :key="item.id">
```

Comment:
Please refer to my comment below regarding opening the InfoPanel to view/edit Link details.

```suggestion
                    <tr v-for="(item, index) in filtered_linkData" :key="item.id" @click="show_infoPanelLink(item.id)">
```